deployment/HDI-Spark/Notebooks/install-livy-sparkmagic-hn.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Update Livy Jars
+#
+
+sudo wget https://ruinliureplhdistorage.blob.core.windows.net/ruinliurepl/livy_jar.zip -O livy_jar.zip
```

Comment:
Should we use a more official sounding storage account name ?

Reply:
@imback82 @rapoth Any suggestions?

Reply:
I have changed the name to `sparkdotnetrepl` and `notebooks`. Let me know if any other ideas. Thanks!